Controller/Checkout/Cart.php

```diff
@@ -31,7 +31,9 @@ public function __construct(
      */
     public function execute()
     {
-        $quoteId = $this->request->getParam('quote_id');
+        $params = $this->request->getParams();
+        $quoteId = $params['quote_id'];
```

Comment:
if `quote_id` isn't exists - we'll have there notice. 
```suggestion
        $quoteId = $params['quote_id'] ?? 0;
```

Reply:
https://github.com/klaviyo/magento2-klaviyo/pull/76